src/main/java/com/FoodyBuddy/Model/Buyer.java

```diff
@@ -0,0 +1,210 @@
+package com.FoodyBuddy.Model;
+
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import org.hibernate.envers.Audited;
+
+@Entity
+@Audited
+@Table(name = "Buyer")
+public class Buyer{
+	
+	@Id
+	@Column(name = "id", nullable = false)
+	@GeneratedValue
+	private int id;
+	
+	@Column(name = "apartment_id", nullable = false)
+	@ManyToOne
+	@JoinColumn(name = "id")
+	private Apartment apartment;
+	
+	@Column(name = "last_modified_by_id")
+	private int lastModifiedById;
+	
+	@Column(name = "name")
+	private String name;
+	
+	@Column(name = "mobile_no")
+	private String mobileNumber;
+	
+	@Column(name = "email")
+	private String email;
+	
+	@Column(name = "flat_no")
+	private String flatNumber;
+	
+	@Temporal(TemporalType.TIMESTAMP)
+	@Column(name = "created_at")
```

Comment:
While this specifies the the database to use Timestamp datatype, we have to also specify default as current_timespamp @junzy @sanjanagyl @SainathB